iron-dropdown.html

```diff
@@ -99,6 +99,26 @@
           },
 
           /**
+           * A pixel value that will be added to the position calculated for the
```

Comment:
This isn't quite correct, since the value will be added in the same direction regardless of the value of `horizontalAlign`. For example, if I `horizontalAlign` to the `"left"`, with a `horizontalOffset` of `10`, I will add `10` to the final horizontal position. If I `horizontalAlign` to the `"right"` instead, it will still add `10` to the final horizontal position. You would have to change it to `-10` to actually change the direction.